javascript/javascript1/week4/getReplay.js

```diff
@@ -0,0 +1,75 @@
+let toDo = [];
+
+function getReply(command) {
+    const commandWords = command.split(' ');
+  if (command.startsWith("hello my name ")) {
+    const name = commandWords[commandWords.length - 1];
+    return " Nice to meet you " + name;
+  } else if (command.startsWith("What is my name")) {
+    return " Your name is Maher";
```

Comment:
hi Zuhair. thank you for your feedback. here in the second question "what is my name " can we make the name dynamic. because we don't have 
name = commandWords[commandWords.length - 1]; in this question.